From ba8e0091c50ed38a2e48452cb80535a3376a186d Mon Sep 17 00:00:00 2001
From: David Kershner <dkershner@habana.ai>
Date: Fri, 13 Jun 2025 05:11:14 +0300
Subject: [PATCH] [SW-226950] oobi: reorder numeric sensor op state

The hl_oobi_pldm_numeric_sensor_op_state structure should use the
MSB field to determine operational state of UNAVAILABLE/DISABLED
instead of LSB.

Change-Id: Ifc74b01e5ce65a5e85117b03441b04be098248d0
Signed-off-by: David Kershner <dkershner@habana.ai>
---

diff --git a/oobi.h b/oobi.h
index 40ab115..2ba9fab 100644
--- a/oobi.h
+++ b/oobi.h
@@ -117,8 +117,8 @@
 } __packed;
 
 struct hl_oobi_pldm_numeric_sensor_op_state {
-	uint16_t sensor_op_state:1; /* oobi_sensor_op_state_t */
 	uint16_t sensor_id:15;
+	uint16_t sensor_op_state:1; /* oobi_sensor_op_state_t */
 } __packed;
 
 struct hl_oobi_pldm_state_sensor_op_state {
